bindings/python/iota_sdk/types/transaction_options.py

```diff
@@ -68,26 +71,38 @@ class TransactionOptions:
     Attributes:
         remainder_value_strategy: The strategy applied for base coin remainders.
         tagged_data_payload: An optional tagged data payload.
-        custom_inputs: If custom inputs are provided only those are used. If also other additional inputs should be used, `mandatory_inputs` should be used instead.
-        mandatory_inputs: Inputs that must be used for the transaction.
+        context_inputs: Transaction context inputs to include.
+        required_inputs: Inputs that must be used for the transaction.
         burn: Specifies what needs to be burned during input selection.
         note: A string attached to the transaction.
         allow_micro_amount: Whether to allow sending a micro amount.
+        allow_additional_input_selection: Whether to allow the selection of additional inputs for this transaction.
+        capabilities: Transaction capabilities.
+        mana_allotments: Mana allotments for the transaction.
+        issuer_id: Optional block issuer to which the transaction will have required mana allotted.
     """
 
     def __init__(self, remainder_value_strategy: Optional[Union[RemainderValueStrategy, RemainderValueStrategyCustomAddress]] = None,
                  tagged_data_payload: Optional[TaggedDataPayload] = None,
-                 custom_inputs: Optional[List[OutputId]] = None,
-                 mandatory_inputs: Optional[List[OutputId]] = None,
+                 context_inputs: Optional[List[ContextInput]] = None,
+                 required_inputs: Optional[List[OutputId]] = None,
                  burn: Optional[Burn] = None,
                  note: Optional[str] = None,
-                 allow_micro_amount: Optional[bool] = None):
+                 allow_micro_amount: Optional[bool] = None,
+                 allow_additional_input_selection: Optional[bool] = None,
+                 capabilities: Optional[HexStr] = None,
+                 mana_allotments: Optional[ManaAllotment] = None,
```

Comment:
Only one ManaAllotment? Plus it also wouldn't serialise as `BTreeMap<AccountId, u64>`?